otterdog/eclipse-tractusx.jsonnet

```
@@ -1547,6 +1547,46 @@ orgs.newOrg('automotive.tractusx', 'eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('tractusx-identityhub') {
```

```suggestion
    orgs.newRepo('ssi-tractusx-identity-hub') {
```
There are quite some ssi related components in Tractus-X now, to all find them in [one search](https://github.com/orgs/eclipse-tractusx/repositories?type=all&q=ssi), adding the `ssi-` would be beneficial.

---

"IdentityHub" is the upstream project name, prepending "ssi" sounds quirky, and adds little value.

---

I don't quite agree for instance the fact that it is located in the eclipse-tractusx GH indicates that it is part of the TX project at the same time I do not feel it would be productive to continue this discussion further, just leave it as is and provide an explanation to this comment https://github.com/eclipse-tractusx/.eclipsefdn/pull/116#issuecomment-2615244792

---

@evegufy  @paullatzelsperger Stupid idea maybe, but what if we call it
identityhub-wallet ?

then people can "find" what is the "new" wallet that is being developed, migrating from MIW to identity hub or is too confusing?

Otherwise I would say we stick with `tractusx-identity-hub`

And if is so important matching the upstream then Call it `tractusx-identityhub` since there there is no space.

but I agree with @paullatzelsperger on that ssi brings not much to it I think, more confusion that is sure.

---

```suggestion
    orgs.newRepo('tractusx-identity-hub') {
```

?

---

In this way we are consistent with the "hub" structure in tractus-x
example:
 https://github.com/eclipse-tractusx/industry-core-hub